code/modules/admin/topic.dm

```diff
@@ -1896,7 +1896,7 @@
 	else if(href_list["cryoplayer"])
 		if(!check_rights(R_ADMIN))	return
 
-		var/mob/M = locate(href_list["cryoplayer"])
+		var/mob/living/carbon/M = locate(href_list["cryoplayer"])

```

Comment:
Will either need a //VOREStation edit, or sent to Polaris. Probably sent to Polaris.